releasenotes/notes/istiod-config-size-bytes.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: release-notes/v2
+kind: feature
+area: networking
+releaseNotes:
```

Comment:
Is there any issue created for this PR, if not, please add the following so it won't miss during the release note generation?
```
issue:
  - 31772
```

Reply:
There was not an issue. This came up in a discussion thread somewere.

Reply:
I see. In that case, adding a PR number as I mentioned will help in release note generation. otherwise it will be miss in the release notes.

Reply:
Can we fix the release notes generator to not skip it instead of making PR authors add things to their PRs?

Reply:
Added issue number to release note, per suggestion.